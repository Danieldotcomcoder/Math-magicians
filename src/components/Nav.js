import React from 'react';
import { Link } from 'react-router-dom';
import './pages.css';

function Nav() {
  return (
    <nav>
      <ul className="Navlist">
        <Link to="/">
          <li id="home">Home</li>
        </Link>
        <Link to="/calculator">
          <li id="calculator">Calculator</li>
        </Link>
        <Link to="/quote">
          <li>Quote</li>
        </Link>
      </ul>
    </nav>

  );
}
export default Nav;
