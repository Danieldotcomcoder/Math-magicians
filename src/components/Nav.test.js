import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Nav from './Nav';

test('render about link', () => {
  const nav = renderer.create(
    <Router>
      <Nav />
    </Router>,
  ).toJSON();
  expect(nav).toMatchSnapshot();
});
