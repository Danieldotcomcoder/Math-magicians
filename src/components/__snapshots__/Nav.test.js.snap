// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render about link 1`] = `
<nav>
  <ul
    className="Navlist"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <li
        id="home"
      >
        Home
      </li>
    </a>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      <li
        id="calculator"
      >
        Calculator
      </li>
    </a>
    <a
      href="/quote"
      onClick={[Function]}
    >
      <li>
        Quote
      </li>
    </a>
  </ul>
</nav>
`;
